examples/trials/enas/README.md

```diff
@@ -0,0 +1,6 @@
+**Run ENAS in NNI**
+===
+
+Now we have a enas version run in nni from our contributors. Thanks our lovely contributors. 

```

Comment:
an enas

Reply:
(Optional) 
How about:
Now we have an enas trial for NNI implemented by community contributors